src/test/ui/mir-dataflow/liveness-projection.rs

```diff
@@ -0,0 +1,32 @@
+#![feature(core_intrinsics, rustc_attrs)]
+
+use std::intrinsics::rustc_peek;
+
+#[rustc_mir(rustc_peek_liveness, stop_after_dataflow)]
+fn foo() {
+    {
+        let mut x: (i32, i32) = (42, 0);
+
+        // Assignment to a projection does not cause `x` to become live
+        unsafe { rustc_peek(x); } //~ ERROR bit not set
```

Comment:
shouldn't this be *after* the assignment? Same with the read below

Reply:
Liveness propagates backwards, not forwards.

Reply:
:facepalm: